packages/touchpoint-ui/src/bidirectional/commandHandler.ts

```diff
@@ -90,7 +88,35 @@ export const commandHandler = (
         }
         break;
       case "custom":
```

Comment:
The handler should be called with all custom events, even the ones that have a handler defined in the new custom action definition method. This is to ensure legacy handlers work as expected in existing projects (even if a custom action will now be sent to the NLU with the new API, it is possible that the handler for it is in here).